tests/utils/test_matrix_operations.py

```diff
@@ -0,0 +1,463 @@
+"""
+Tests for matrix operations for zeroing rows/columns, efficient slicing, stacking,
+merging and construction from arrays.
+"""
+import numpy as np
+import pytest
+import scipy.sparse as sps
+
+from porepy import matrix_operations
+from porepy.applications.test_utils.arrays import compare_matrices
+
+
+def test_zero_1_column():
```

Comment:
@Yuriyzabegaev Have you considered whether it is meaningful to parametrize these tests? I could not decide myself.

Reply:
@keileg I thought about it and came up with this. Imo, parametrization is meaningful when it makes the code simpler to read and understand. Here, it is clearly written, and abusing parametrization could make it way harder. Imagine two separate columns of parametrization: results and expected results, each with numpy arrays or matrices. This scared me, and it will be more than two columns in practice...

Reply:
agreed.